src/containers/shared/components/TxToken.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import '../css/txlabel.scss'
+
+interface Props {
+  tx: any
+}
+
+// We should consider moving this logic to each individual parser. This would give us more customizability.
+function getTokenPair(
+  type: string,
+  fee: number,
+  amount: { currency: string; amount: number },
+  amount2: { currency: string; amount: number },
+) {
+  if (type === 'AMMBid' || type === 'AMMVote') {
```

Comment:
Could it be an LP token if it was part of a payment between two other parties and the tx is returned in `account_tx`?

Reply:
yeah i am talking about the potental for someone to create offers for LPTokens that dont interact with the AMM directly.  Those transactions should be returned in account_tx.  What should we show for those?

Reply:
do you have an example transaction? what's the transaction type? 

this might be more of a product question.

Reply:
return LP by default